static/js/util/util.js

```diff
@@ -378,7 +378,9 @@ export function formatPrice(price: ?string|number|Decimal): string {
   if (price === null || price === undefined) {
     return '';
   } else {
-    return `$${price}`;
+    const formatedPrice = Decimal(price).todp(2, Decimal.ROUND_DOWN).toFixed(2);
```

Comment:
According to the documentation the default rounding mode is `ROUND_HALF_UP` which seems more appropriate for this case. The problem with `ROUND_DOWN` is that `10.619` will become `10.61` instead of `10.62` which is closer. Why are we overriding the default rounding setting here?

Also, `toFixed(...)` will round the value. We shouldn't need a separate `round(...)` call here